packages/grid/x-data-grid/src/index.ts

```diff
@@ -1,6 +1,10 @@
 export * from '../../_modules_';
 export * from './DataGrid';
-export type { DataGridProps } from '../../_modules_/grid/models/props/DataGridProps';
+export type {
+  DataGridProps,
+  GridExperimentalFeatures,
+} from '../../_modules_/grid/models/props/DataGridProps';
 export * from './useDataGridComponent';
-export { MAX_PAGE_SIZE } from './useDataGridProps';
+export { MAX_PAGE_SIZE, DATA_GRID_PROPS_DEFAULT_VALUES } from './useDataGridProps';
 export * from './typeOverload';
+export * from './private';
```

Comment:
Because with the current bundling strategy, `import from '@mui/x-data-grid/private'` is impossible since we are only generating single-file bundles.

But yes the goal is to change that before the release, when migrating to the core bundling strategy.